src/Supervisor/Console/RestartHorizon.php

```diff
@@ -27,33 +29,34 @@ class RestartHorizon extends Command
      */
     public function handle(): int
     {
-        // First Check if horizon is installed.
         try {
-            $horizon = app(\Laravel\Horizon\Contracts\MasterSupervisorRepository::class);
```

Comment:
I guess doing this leafing to crash qgole command if horizon not installed, that's why, i added it as path here

Reply:
@ralphjsmit

Reply:
Are you sure that this import is causing a crash, different then from the inline classname? Is it crashing for you?

Reply:
i didn't test it, to be honset, but i guess if import doens't exist in system, it would crash.
not sure,
maybe i'm wrong.

Reply:
In that case I think I'm going leave it like this, because as far as I know it should work the same with or without import. If you really are experiencing some error, ping me and I'll update it.